packages/utils/src/bulk-insert-patients.ts

```diff
@@ -181,6 +188,7 @@ function normalizeState(state: string | undefined): USState {
 }
 
 const mapCSVPatientToMetriportPatient = (csvPatient: {
+  id: string | undefined;
```

Comment:
can we also add a column `externalId` and use `id ?? externalId` below? One of our cxs sent that info on a column named `externalId`

Reply:
I'll add that. But going forwards, we should update the template to be consistent